docker/install-deps

```diff
@@ -81,6 +81,9 @@ if [ -f /usr/bin/yum ]; then
         PKGLIST+=" devtoolset-7-gcc-c++ devtoolset-7-binutils cmake3"
         PKGLIST+=" devtoolset-7-libasan-devel devtoolset-7-libubsan-devel"
         PKGLIST+=" devtoolset-7-valgrind devtoolset-7-valgrind-devel"
```

Comment:
devtoolset-7 is redundant then?
Also, the way it all looks structured, devtoolset-2 is always installed, regardless of it is needed or not?

Reply:
We add  all changes only in docker/install-deps for branch 8.0  and then use it to build docker images which are used for all branches that is why we need to install all these rpms. devtoolset-2  is used for Centos 6/i386 .

Reply:
We add  all changes only in docker/install-deps for branch 8.0  and then use it to build docker images which are used for all branches that is why we need to install all these rpms. devtoolset-2  is used for Centos 6/i386 .